src/vstart.sh

```diff
@@ -906,6 +842,67 @@ fi
 
 if [ $CEPH_NUM_MON -gt 0 ]; then
     start_mon
+
+    echo Populating config ...
+    cat <<EOF | $CEPH_BIN/ceph -c ceph.conf config assimilate-conf -i -
```

Comment:
Note: this breaks clusters started w/ ```mstart.sh``` since you assume the cluster config is located at ```ceph.conf```, which in turn breaks ```qa/workunits/rbd/rbd_mirror.sh``` on a dev box.